sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/optimizer/Optimizer.scala

```diff
@@ -143,6 +143,7 @@ abstract class Optimizer(catalogManager: CatalogManager)
       EliminateResolvedHint,
       EliminateSubqueryAliases,
       EliminateView,
+      EliminateDataFrameTempViews,
```

Comment:
Do we need to put this rule here? `EliminateView` should remove all `View` wrappers.